edenai_apis/apis/microsoft/microsoft_video_api.py

```
@@ -37,70 +67,70 @@ def video__generation_async__launch_job(
-            generation_id = generations[0].get("id")
-            video_url = f"{base_url}openai/v1/video/generations/{generation_id}/content/video?api-version=preview"
-            video_response = requests.get(video_url, headers={"Authorization": api_key})
-            base64_encoded_string = base64.b64encode(video_response.content).decode(
-                "utf-8"
+                message=_extract_error_message(original_response),
+                code=response.status_code,
+            ) from e
+
+        status = original_response.get("status")
+        if status == "completed":
+            response = requests.get(
```

Ruff (0.15.6)

[error] Probable use of `requests` call without timeout

(S113)